qui/updater.py

```diff
@@ -12,6 +12,10 @@
 from gi.repository import Gtk, Gdk, GObject, Gio  # isort:skip
 from qubesadmin import Qubes
 
+import locale
+from locale import gettext as _
+locale.bindtextdomain("desktop-linux-manager", "/usr/locales/")
+locale.textdomain('desktop-linux-manager')
```

Comment:
Please add a comment why this is different than the other files (because of glade/gtkbuilder).